clientconn_test.go

```diff
@@ -344,7 +344,7 @@ func TestDialTimeout(t *testing.T) {
 	}
 }
 
-func TestTLSDialTimeout(t *testing.T) {
```

Comment:
How about `TestWithTransportCredentialsTLS`, and *this* one can use `DialContext()` and a context with a timeout?

Reply:
Done!